docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+redis:
```

Comment:
As discussed on one of our meeting, we don't need stand-alone version now. So the current solution connects to one of our existing solution.